components/eamxx/CMakeLists.txt

```diff
@@ -1,7 +1,18 @@
+####################################################################
```

Comment:
The changes here are mostly a reorg of the CMakeLists, to group together settings that are conceptually related, to make it easier to navigate the file.

Reply:
There are also a couple of vars that changed name, and I removed some pointless DEFAULT_xyz vars, using their value directly in the main cache var declaration.